src/Misc/layoutbin/en-US/strings.json

```diff
@@ -277,6 +277,13 @@
   "DirExpireLimit": "Directory expiration limit: {0} days.",
   "DiscoverBuildDir": "Discover stale build directories that haven't been used for more than {0} days.",
   "DiscoverReleaseDir": "Discover stale release directories that haven't been used for more than {0} days.",
+  "DockerContainerPersistentFailures": [
+    "❌ The current step has failed inside the Docker container with exit code {0}.",
+    "If this issue persists, consider the following troubleshooting steps:",
+    "\t- 🛜 Check the container and host network configurations;",
+    "\t- ⚒️ Rebuild the Docker image if feasible.",
```

Comment:
these troubleshooting steps are too generic to be helpful - I think it's better to defer the steps to the documentation instead